Tasks/ANT/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 30
+        "Patch": 31
```

Comment:
I also bumped the patch version for every task. It was just easier that way. Technically there were 11 that don't need to be bumped. However, I need to make a follow-up check-in in the master branch, to bump the versions twice. So it was easier to just bump all tasks rather than track which ones should be bumped and which shouldn't - since almost all need to be bumped anyway.